### PR TITLE
21898 - Remove AUTH_WEB_TOKEN_CONFIRM_PATH

### PR DESCRIPTION
**Pull request by seeker25**
https://github.com/bcgov/entity/issues/21898

We saw an issue where =null was being put into this environment variable and would break the code in the PR.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Review by Jxio — APPROVED** [review]

File: auth-api/src/auth_api/services/affiliation_invitation.py [hunk 1]
```diff
@@ -279,8 +278,7 @@ def create_affiliation_invitation(affiliation_invitation_info: Dict,
         AffiliationInvitation\
             .send_affiliation_invitation(affiliation_invitation=affiliation_invitation,
                                          business_name=business['business']['legalName'],
-                                         app_url=AffiliationInvitation._get_app_url(invitation_origin,
-                                                                                    context_path),
+                                         app_url=invitation_origin + '/',
```
Comment (seeker25) [inline auth-api/src/auth_api/services/affiliation_invitation.py:281]: rather have extra slashes to be safe, browser can handle it

File: auth-api/src/auth_api/services/affiliation_invitation.py [hunk 2]
```diff
@@ -344,8 +341,7 @@ def update_affiliation_invitation(self, user, invitation_origin, affiliation_inv
             AffiliationInvitation\
                 .send_affiliation_invitation(affiliation_invitation=invitation,
                                              business_name=business['business']['legalName'],
-                                             app_url=AffiliationInvitation._get_app_url(invitation_origin,
-                                                                                        context_path),
+                                             app_url=invitation_origin + '/',
```
Comment (seeker25) [inline auth-api/src/auth_api/services/affiliation_invitation.py:344]: rather have extra slashes to be safe, browser can handle it

File: auth-api/src/auth_api/services/invitation.py [hunk 3]
```diff
@@ -115,7 +114,7 @@ def create_invitation(invitation_info: Dict, user, invitation_origin, **kwargs):
         invitation.login_source = mandatory_login_source
         invitation.save()
         Invitation.send_invitation(invitation, org_name, org.id, user.as_dict(),
-                                   f'{invitation_origin}/{context_path}', mandatory_login_source,
+                                   f'{invitation_origin}/', mandatory_login_source,
```
Comment (seeker25) [inline auth-api/src/auth_api/services/invitation.py:117]: rather have extra slashes to be safe, browser can handle it

File: auth-api/src/auth_api/services/invitation.py [hunk 4]
```diff
@@ -162,7 +160,7 @@ def update_invitation(self, user, invitation_origin):
         updated_invitation = self._model.update_invitation_as_retried()
         org_name = OrgModel.find_by_org_id(self._model.membership[0].org_id).name
         Invitation.send_invitation(updated_invitation, org_name, self._model.membership[0].org_id, user.as_dict(),
-                                   f'{invitation_origin}/{context_path}', self._model.login_source,
+                                   f'{invitation_origin}/', self._model.login_source,
```
Comment (seeker25) [inline auth-api/src/auth_api/services/invitation.py:163]: rather have extra slashes to be safe, browser can handle it

File: auth-api/src/auth_api/services/invitation.py [hunk 5]
```diff
@@ -367,7 +364,7 @@ def notify_admin(user, invitation_id, membership_id, invitation_origin):
 
         if admin_emails != '':
             Invitation.send_admin_notification(user.as_dict(),
-                                               f'{invitation_origin}/{context_path}',
+                                               f'{invitation_origin}/',
```
Comment (seeker25) [inline auth-api/src/auth_api/services/invitation.py:367]: rather have extra slashes to be safe, browser can handle it

File: auth-api/src/auth_api/services/membership.py [hunk 6]
```diff
@@ -163,8 +163,7 @@ def send_notification_to_member(self, origin_url, notification_type):
             current_app.logger.error('No user contact record for user id %s', self._model.user_id)
             current_app.logger.error('<send_notification_to_member failed')
         recipient = self._model.user.contacts[0].contact.email
-        context_path = CONFIG.AUTH_WEB_TOKEN_CONFIRM_PATH
-        app_url = f'{origin_url}/{context_path}'
+        app_url = f'{origin_url}/'
```
Comment (seeker25) [inline auth-api/src/auth_api/services/membership.py:166]: rather have extra slashes to be safe, browser can handle it

File: auth-api/src/auth_api/services/org.py [hunk 7]
```diff
@@ -858,7 +858,7 @@ def send_staff_review_account_reminder(relationship_id,
         task = TaskModel.find_by_task_relationship_id(task_relationship_type=task_relationship_type,
                                                       relationship_id=relationship_id)
         context_path = f'review-account/{task.id}'
-        app_url = f"{g.get('origin_url', '')}/{current_app.config.get('AUTH_WEB_TOKEN_CONFIRM_PATH')}"
+        app_url = f"{g.get('origin_url', '')}/"
```
Comment (seeker25) [inline auth-api/src/auth_api/services/org.py:861]: rather have extra slashes to be safe, browser can handle it

File: auth-api/src/auth_api/services/user.py [hunk 8]
```diff
@@ -234,7 +234,7 @@ def delete_otp_for_user(user_name, origin_url: str = None):
     def send_otp_authenticator_reset_notification(recipient_email, origin_url, org_id):
         """Send Authenticator reset notification to the user."""
         current_app.logger.debug('<send_otp_authenticator_reset_notification')
-        app_url = f"{origin_url}/{current_app.config.get('AUTH_WEB_TOKEN_CONFIRM_PATH')}"
+        app_url = f'{origin_url}/'
```
Comment (seeker25) [inline auth-api/src/auth_api/services/user.py:237]: rather have extra slashes to be safe, browser can handle it

File: auth-api/src/auth_api/services/org.py [hunk 9]
```diff
@@ -913,7 +913,7 @@ def send_approved_rejected_govm_govn_notification(receipt_admin_email, org_name,
             notification_type = QueueMessageTypes.GOVM_REJECTED_NOTIFICATION.value
         else:
             return  # Don't send mail for any other status change
-        app_url = f"{origin_url}/{current_app.config.get('AUTH_WEB_TOKEN_CONFIRM_PATH')}"
+        app_url = f'{origin_url}/'
```
Comment (seeker25) [inline auth-api/src/auth_api/services/org.py:916]: rather have extra slashes to be safe, browser can handle it

File: auth-api/src/auth_api/services/org.py [hunk 10]
```diff
@@ -887,7 +887,7 @@ def send_approved_rejected_notification(receipt_admin_emails, org_name, org_id,
             notification_type = QueueMessageTypes.NON_BCSC_ORG_REJECTED_NOTIFICATION.value
         else:
             return  # Don't send mail for any other status change
-        app_url = f"{origin_url}/{current_app.config.get('AUTH_WEB_TOKEN_CONFIRM_PATH')}"
+        app_url = f'{origin_url}/'
```
Comment (seeker25) [inline auth-api/src/auth_api/services/org.py:890]: rather have extra slashes to be safe, browser can handle it

**Pull request merged**